apache/yunikorn-site
# [YUNIKORN-2256] Fix link error occurs when clicking link on page with trailing slash in the URL

### What is this PR for?
I've observed that many links in the Chinese documentation can't be converted to markdown relative file paths because some documents are not yet translated into Chinese. The task of changing all links to markdown file paths is currently not possible.
In the pull request:
1. setting trailingSlash to true
2. fixing any links that become invalid due to this setting.

### What type of PR is it?

### Todos
N/A

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2256

### How should this be tested?
covered by `yarn build` in docker images, in docusaurus, `yarn build` will check if there is any broken links, if there is any, error occurs.

### Screenshots (if appropriate)
N/A

### Questions:
N/A
